### PR TITLE
Disable dnf playground command

### PR DESCRIPTION
**Pull request by schlupov**
= changelog =
msg: playground command doesn't work
type: bugfix
related: https://bugzilla.redhat.com/show_bug.cgi?id=1955907

### CODE REVIEW

**Comment by praiskup** [comment]
Otherwise, I think it does what it should.

File: plugins/copr.py [hunk 1]
```diff
@@ -681,6 +683,7 @@ def set_argparser(parser):
                             choices=['enable', 'disable', 'upgrade'])
 
     def run(self):
+        raise dnf.exceptions.Error("The playground command is currently disabled")
```
[the remark this answers is not in the document]
Comment (praiskup) [reply plugins/copr.py:686]: Or "Playground feature is temporarily unavailable"?

[the remark this answers is not in the document]
Comment (schlupov) [reply plugins/copr.py:686]: Sounds better, I'll change it.

[the remark this answers is not in the document]
Comment (schlupov) [reply plugins/copr.py]: I like that - Playground is temporarily unsupported

**Comment by praiskup** [comment]
Looks good to me.  Thanks!

**Pull request merged**